add-blobs.sh

```diff
@@ -6,10 +6,10 @@ mkdir -p .downloads
 
 cd .downloads
 
-ES_VERSION=6.7.1
+ES_VERSION=7.0.0
 
 if [ ! -f ${DIR}/blobs/elasticsearch/elasticsearch-${ES_VERSION}.tar.gz ];then
-    curl -L -J -o elasticsearch-${ES_VERSION}.tar.gz https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-oss-${ES_VERSION}.tar.gz
+    curl -L -J -o elasticsearch-${ES_VERSION}.tar.gz https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-oss-${ES_VERSION}-linux-x86_64.tar.gz
```

Comment:
This link contains JDK. We don't need it.
https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-7.0.0-no-jdk-linux-x86_64.tar.gz is the right answer.